src/context/CardsContext.tsx

```diff
@@ -240,6 +250,24 @@ const CardsContextProvider = ({ children }: CardContextProps) => {
     return data?.data;
   }, []);
 
+  const getOnchainTransfers = useCallback(async (params: GetOnchainTransfersParams) => {
+    const { address, tokenAddress, fromDate, skipSettlementTransfers } = params;
```

Comment:
There's one place in the code where I could destructure the params, but other than this exception, I'd suggest to do it right away

```suggestion
  const getOnchainTransfers = useCallback(async ({ address, tokenAddress, fromDate, skipSettlementTransfers }: GetOnchainTransfersParams) => {
```